nrrd/writer.py

```diff
@@ -124,6 +124,10 @@ def write(filename, data, header={}, detached_header=False, custom_field_map=Non
     custom_field_map : :class:`dict` (:class:`str`, :class:`str`), optional
         Dictionary used for parsing custom field types where the key is the custom field name and the value is a
         string identifying datatype for the custom field.
+    compression_level : int
```

Comment:
Use :class:`int` here just so Sphinx will link to it. Just to stay consistent with the other parameters.

Reply:
Ok, for consistency. But linking docs for native types is excessive IMHO, and it makes the docstrings ugly outside of sphinx.